JS/Anna_G/TaskOfBookStore/dist/books.css

```diff
@@ -0,0 +1,145 @@
+.center{
```

Comment:
all those 'dist/*' files should be moved to sources. This folder is generates when build starts so in case of rebuild those files will be removed